.travis.yml

```diff
@@ -15,12 +15,13 @@ install:
 
 script: rspec
 
-deploy: 
-  provider: script
-  script: .travis/deploy.sh
-  on: 
-    tags: true
-    condition: '"$($TRAVIS_BUILD_DIR/.travis/is_valid_tag.rb $TRAVIS_TAG)" == "" && $? == 0'
+deploy:
+  provider: rubygems
+  api_key: $RUBYGEMS_SNOWPLOW_API_KEY
+  gemspec: iglu-ruby-client.gemspec
+  on:
+   tags: true
+
```

Comment:
Please, add a `condition` from previous version as well (and `.travis/is_valid_tag.rb` as it is required), I should have warn you about it. 

Without `condition` pushing could result in a quite serious issue when we would push `0.3.0.beta` tag, but `0.3.0` by mistake remains in source code - as a result `0.3.0` will be published to RubyGems, which is in fact should have been `0.3.0.beta` (by our intentions, e.g. git tag). And we cannot delete releases from RubyGems, which means we'll leave invalid release on RubyGems.

We have this situation right now: `0.2.0.beta` is on RubyGems, but in source code there's `0.2.0.beta.2`. It's quite ok for beta-releases, but should never happen for final releases. `condition` makes us safe here.